packages/dev/core/src/Events/deviceInputEvents.ts

```diff
@@ -101,6 +103,11 @@ export interface IKeyboardEvent extends IUIEvent {
  */
 export interface IMouseEvent extends IUIEvent {
     // Properties

```

Comment:
I would recommend removing these lines. It's confusing and unnecessary.
```suggestion
```

Reply:
Removed